pyracf/common/alter_operation_error.py

```diff
@@ -0,0 +1,24 @@
+"""Exception to use when Alter operation would add a new profile."""
+
+
+class AlterOperationError(Exception):
+    """
```

Comment:
I suggest changing the docstring to this instead since it makes it more clear why the exception gets raised.

```python
    """
    Raised when a profile cannot be altered because it does not exist.
    """
```

Reply:
Agreed. Change made.